docs/deploy-development-rp.md

```diff
@@ -76,42 +76,109 @@
      >/dev/null
    ```
 
+## Preparation to Create Cluster:
+
+1. Update the Address Space of "rp-vnet" to allow for creation of a new VPN. You should be able to do this at: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscription-id>/resourceGroups/<aro-shared-rp-rg>/providers/Microsoft.Network/virtualNetworks/rp-vnet/addressSpace. See the table below for what we did.
+
+| Address Space | Address Range         | Address Count |
+| ------------- | --------------------- | ------------- |
+| 10.0.0.0/24   | 10.0.0.0 - 10.0.0.255 | 256 |
+| 10.1.0.0/24   | 10.1.0.0 - 10.1.0.255 | 256 |
+
+1. Create a new "Virtual Network Gateway (Gateway type: VPN)" in the Azure Portal manually. This needs to be configured to the "Virtual Network" named "rp-vnet" which will already existing in the shared RP's resource group. This new VPN will allow the local ARO-RP to connect to the to the existing "rp-vnet" to create a cluster. 
+
+1. Configure the new "rp-vnet" VPN with the same public certificate used for the existing dev-vpn. This is done at: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscription-id>/resourceGroups/<aro-shared-rp-rg>/providers/Microsoft.Network/virtualNetworkGateways/rp-vnet/pointtositeconfiguration. You can simply copy the info from: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscription-id>/resourceGroups/<aro-shared-rp-rg>/providers/Microsoft.Network/virtualNetworkGateways/dev-vpn/pointtositeconfiguration.
+
+1. Connect to the "rp-vnet" VPN created above. You can use openvpn or the azure vpn client, both have worked fine in our testing. 
+  1. Go to Point-to-Site Configuration for "rp-vnet" (https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscription-id>/resourceGroups/<aro-shared-rp-rg>/providers/Microsoft.Network/virtualNetworkGateways/rp-vnet/pointtositeconfiguration) and download the VPN Client Certificate to your local environment. You can extract the zip file anywhere you would like, but we put it under the "secrets" folder because that is where the ARO-RP secrets reside.
```

Comment:
I believe the links are meant to be a generalized link instead of to a specific subscription or RG, but the rendering is stripping the content in the angle brackets as if it's a HTML tag.

Reply:
Yes, the links are meant to be general to give the user an idea of what to expect, not be a hard-link to click on for navigation. How about offsetting with backticks to signify that?

Reply:
@cadenmarchese resolving but please advise if you'd like further cleanup. I went with back ticks to offset the example urls.